jsxc.lib.js

```diff
@@ -223,9 +223,15 @@ var jsxc;
             lang = jsxc.options.defaultLang;
          }
 
-         // set language
-         jsxc.l = jsxc.l10n.en;
-         $.extend(jsxc.l, jsxc.l10n[lang]);
+         // initialize i18n translator
+         $.i18n.init({
+           lng: lang,
+           fallbackLng: 'en',
+           resStore: I18next,
+           // use localStorage and set expiration to a day
+           useLocalStorage: true,
```

Comment:
If we use resStore, we don't have to use the local storage, or not? Why are we not using the ajax loading?